### PR TITLE
Let users define labels when creating routes

### PR DESCRIPTION
**Pull request by spadgett**
Fixes https://github.com/openshift/origin/issues/6095

openshift_web_console

@jwforres PTAL

### CODE REVIEW

**Comment by jwforres** [comment]
I can't decide on the system labels here, I see the argument for it, but at the same time we don't let you remove the system labels.

**Comment by spadgett** [comment]
I don't disagree, but I think it's a problem in general with how we approach system labels. What if I don't want an `app` label when I deploy my template for instance?

**Comment by jwforres** [comment]
i think the difference that bothers me here is that in other cases they are really system labels, in this case its not just copying the system labels like the app label, its copying all the labels from the service, some of which may have been user defined

**Comment by spadgett** [comment]
I could prefill them as user-defined labels, but it gets tricky if the user changes services in the dropdown. Or maybe a "Copy from Service" link?

**Comment by jwforres** [comment]
copy from service link seems reasonable, then its easy without being forced into it

**Comment by spadgett** [comment]
Updated with a "Copy Service Labels" link.

File: app/views/create-route.html [hunk 1]
```diff
@@ -27,6 +27,13 @@
                             services="services"
                             show-name-input="true">
                         </osc-routing>
+                        <label-editor
+                            labels="labels"
+                            expand="true"
+                            can-toggle="false"
+                            help-text="Labels for this route.">
+                        </label-editor>
+                        <a href="" ng-click="copyServiceLabels()">Copy service labels</a>
```
Comment (jwforres) [inline app/views/create-route.html]: headline case

Comment (spadgett) [inline app/views/create-route.html]: Right above this link is "Add label" (at least until #978 merges)

Comment (spadgett) [inline app/views/create-route.html]: Let's merge the other one first, and I'll rebase this one. About to push updates for #978

**Comment by spadgett** [comment]
Updated to headline case.

[merge]

**Pull request merged**